formats/grok.go

```diff
@@ -0,0 +1,74 @@
+package formats
+
+import (
+	"fmt"
+	"github.com/turbot/tailpipe-plugin-sdk/constants"
+	formats2 "github.com/turbot/tailpipe-plugin-sdk/formats"
```

Comment:
Should this be aliased to `sdkformats` to keep consistent with `core/plugin.go` ?

Reply:
good catch